pinc/showavailablebooks.inc

```diff
@@ -118,16 +118,15 @@ function show_projects_for_smooth_reading()
     echo "\n<h2>$title</h2>";
 
     $content = 'smoothreading';
-    $xml_rss_images = "
+    $rss_image = "
         <a href='$code_url/feeds/backend.php?content=$content'>
-        <img src='$code_url/graphics/xml.gif' width='36' height='14' alt='[XML]'></a>
-        <a href='$code_url/feeds/backend.php?content=$content&amp;type=rss'>
-        <img src='$code_url/graphics/rss.gif' width='36' height='14' alt='[RSS]'></a>
```

Comment:
We can delete `graphics/rss.gif` in this PR too. Need to leave the XML one around for a bit though since the Teams and Members code uses it.

Reply:
Done.